src/Core/ConstraintValidator/CustomerNameValidator.php

```diff
@@ -38,7 +38,7 @@
 class CustomerNameValidator extends ConstraintValidator
 {
     const PATTERN_NAME = '/^(?!\s*$)(?:[^0-9!<>,;?=+()\/\\\\@#"°*`{}_^$%:¤\[\]|\.。]|[。\.](?:\s|$))*$/u';
-    const PATTERN_DOT_SPACED = '/[\.。](\s{1}[^\ ]|$)/';
+    const PATTERN_DOT_SPACED = '/(?!^)[\.。](\s{1}[^\ ]|$)/';
```

Comment:
Can we have unit test for this class.
I'm afraid it has never been tested correctly :sweat_smile: 
```
F  . . . 。 Lefevre
```
Look valid